test/unit/api/c/capi_types_black.cpp

```diff
@@ -80,5 +80,21 @@ TEST_F(TestCApiBlackTypes, printEnum)
   ASSERT_EQ(ss.str(), expected);
 }
 
+TEST_F(TestCApiBlackTypes, option_category_to_string)
+{
+  ASSERT_DEATH(
+      cvc5_modes_option_category_to_string(static_cast<Cvc5OptionCategory>(-5)),
+      "invalid option category");
+
+  for (int32_t c = static_cast<int32_t>(CVC5_OPTION_CATEGORY_REGULAR);
+       c < static_cast<int32_t>(CVC5_OPTION_CATEGORY_LAST);
+       ++c)
+  {
+    std::string str = cvc5_modes_option_category_to_string(
+        static_cast<Cvc5OptionCategory>(c));
+    ASSERT_NE(str, "?");
+  }
```

Comment:
I understand this is an easy way to test that the function "works" for all the values, and it's also future-proof in the sense that it doesn't require updating if more enum values are added later. However, it would be more robust to print all enum values as a string and check against an expected string with `ASSERT_EQ`, as we do for the other enum types. Please consider this option.

Reply:
You are right, it's a single value for the other enums. I suppose my point was that testing for "not equal to ?" is a weaker check than explicitly verifying equality with an expected value. That’s the part I didn't really like. On the other hand, I liked that you tested all possible values. Since this isn't a critical issue, we can think it over a bit more and handle the refactor later.

Reply:
Right, just for the record, we do have tests like this here: https://github.com/cvc5/cvc5/pull/12011/files#diff-42d8f2a8cf52e1df4f2c47c9f2082be960586803db1364201b028fec81557b02R49